Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml

```diff
@@ -1589,12 +1589,119 @@ script:
       required: true
     description: Delete data acquisition.
     name: fireeye-hx-delete-data-acquisition
+  - name: fireeye-hx-delete-indicator-condition
+    arguments:
+    - name: category
+      required: true
+      description: The indicator category.
+    - name: indicator_name
+      required: true
+      description: The indicator name. The name of the indicator. Use the `uri_name` value.
```

Comment:
The indicator name? or The name of the indicator?

Reply:
![image](https://user-images.githubusercontent.com/81086590/159172492-ba617f1d-ea73-4338-994d-7752441e1eda.png)